src/utils.ts

```diff
@@ -1,28 +1,13 @@
-export const paramToDegrees = (
-  paramMin: number,
-  paramMax: number,
-  startAngle: number,
-  endAngle: number,
-  paramValue: number
-) => {
-  return Math.floor(
-    ((paramValue - paramMin) * (endAngle - startAngle)) /
-      (paramMax - paramMin) +
-      startAngle
-  );
-};
-
-export const degreesToParam = (
-  paramMin: number,
-  paramMax: number,
-  startAngle: number,
-  endAngle: number,
-  degrees: number
```

Comment:
This is the new generic rangeMap function, which does the job of both paramToDegrees and degreesToParam generically. It just takes two ranges and a value and maps the value from one range to another.